### ISSUE TITLE
About image processing

### ISSUE THREAD

**Issue by AlgoHunt** [Tencent/tencent-ml-images#25]
Hi

I have some question about image normalization. as I notice the preprocess way is not like some standard resnet .

if we have an image in rgb order with dataformat of int8 ranging from 0~255. so are we suppose to do the nomalization as below:

image = ((image/255) -0.5 ) * 2

Thanks

**Comment by mylyu** [comment]
> Hi
> 
> I have some question about image normalization. as I notice the preprocess way is not like some standard resnet .
> 
> if we have an image in rgb order with dataformat of int8 ranging from 0~255. so are we suppose to do the nomalization as below:
> 
> image = ((image/255) -0.5 ) * 2
> 
> Thanks

I think so. I have tested this.

**Comment by wubaoyuan** [comment]
@AlgoHunt Yes, please refer to Line 165 in image_preprocessing.py. We normalize the image to [-1, 1]. 
@mylyu  Thanks.

**Comment by bobopit** [comment]
I download the openImage dataset from urls, the size of the whole dataset is about 14T, and the pixel of every image in  openImage >4k.  How should i scale it? or CenterCrop? in order to save storage
